features/machine/machine.feature

```diff
@@ -225,3 +225,32 @@ Feature: Machine features testing
       | name     | <%= cb.noderef_name %> |
     Then the step should succeed
     And the output should match "node-role.kubernetes.io/infra="
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-32269
+  @admin
+  @destructive
+  Scenario Outline: Implement defaulting machineset values for AWS/GCP/Azure/Vsphere
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+    And admin ensures "default-valued-<path>" machineset is deleted after scenario
```

Comment:
Please put the delete step after the create step

Reply:
thanks , made a new commit ..